Source: http://git.savannah.gnu.org/gitweb/?p=libtasn1.git;a=patch;h=62852408cf7e7afafcb9baa01cf7b2740cc46680
Upstream: yes
Reason: Fix out of tree build

From 62852408cf7e7afafcb9baa01cf7b2740cc46680 Mon Sep 17 00:00:00 2001
From: Thierry Reding <thierry.reding@avionic-design.de>
Date: Thu, 1 Nov 2012 09:13:54 +0100
Subject: [PATCH] Fix out of tree build

This fixes a few occurrences where files are incorrectly referenced from
the build directory instead of the source directory, therefore causing
breakage when building out of tree.

Signed-off-by: Thierry Reding <thierry.reding@avionic-design.de>
Signed-off-by: Nikos Mavrogiannopoulos <nmav@gnutls.org>
---
 doc/Makefile.am |    4 ++--
 lib/Makefile.am |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 4cba357..037890b 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -136,8 +136,8 @@ $(gdoc_MANS) $(gdoc_TEXINFOS):
 GDOC_SRC = $(top_srcdir)/lib/decoding.c $(top_srcdir)/lib/errors.c	\
 	$(top_srcdir)/lib/parser_aux.c $(top_srcdir)/lib/coding.c	\
 	$(top_srcdir)/lib/element.c $(top_srcdir)/lib/gstr.c		\
-	$(top_srcdir)/lib/structure.c $(top_builddir)/lib/ASN1.c	\
-	$(top_builddir)/lib/version.c
+	$(top_srcdir)/lib/structure.c $(top_srcdir)/lib/ASN1.c		\
+	$(top_srcdir)/lib/version.c
 
 update-makefile:
 	@MANS=""; \
diff --git a/lib/Makefile.am b/lib/Makefile.am
index 044f72d..2ea4c09 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -19,7 +19,7 @@
 SUBDIRS = gllib
 
 AM_CFLAGS = $(WARN_CFLAGS) $(WERROR_CFLAGS) $(CFLAG_VISIBILITY)
-AM_CPPFLAGS = -I$(builddir)/gllib -DASN1_BUILDING
+AM_CPPFLAGS = -I$(srcdir)/gllib -DASN1_BUILDING
 
 include_HEADERS = libtasn1.h
 
-- 
1.7.2.5

